scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/Common.scala

```diff
@@ -54,23 +54,13 @@ object Common {
   def unrollTaps(step: FlowStep[JobConf]): Seq[Tap[_, _, _]] =
     unrollTaps(step.getSources.asScala.toSeq)
 
-  /**
-   * Get the total size of the input paths, which may contain a glob
-   * pattern in its path, so we must be ready to handle that case.
-   */
-  def inputSizes(step: FlowStep[JobConf]): Seq[(Path, Long)] = {
+  def inputSizes(step: FlowStep[JobConf]): Seq[(String, Long)] = {
     val conf = step.getConfig
-
-    FileInputFormat
-      .getInputPaths(conf)
-      .map { path =>
-        val fs = path.getFileSystem(conf)
-        val size = fs.globStatus(path)
-          .map(status => fs.getContentSummary(status.getPath).getLength)
-          .sum
-
-        path -> size
-      }
+    unrollTaps(step) flatMap {
```

Comment:
nit: keep '.' i.e. unrollTaps(step).flatMap to be consistent with rest of the file and twitter coding conventions.